packages/core/src/core.ts

```diff
@@ -121,8 +128,8 @@ export function debug(message: string): void {
  * Adds an error issue
  * @param message error issue message
  */
-export function error(message: string): void {
-  issue('error', message)
+export function error(message: string | Error): void {
```

Comment:
should `warning` allow Error also?

Reply:
I enabled this for warning. I don't think we need to enable it for info or debug at this time, but if we get feature requests for it we can.